templates/article/article.js

```diff
@@ -0,0 +1,15 @@
+function addPressReleaseDate(main, doc) {
+  const prHead = main.querySelector('div > div > h1');
+  const prDate = doc.querySelector('meta[name="publish-date"]');
+  // this should only fire on press-release pages
+  if (window.location.href.indexOf('/news-and-stories/press-releases/') !== -1 && prHead && prDate) {
```

Comment:
nitpick, easier on the eyes: 

```suggestion
  if (window.location.href.includes('/news-and-stories/press-releases/') && prHead && prDate) {
```